app-nexus-network/app-nexus-network-htb.js

```diff
@@ -184,6 +184,9 @@ function AppNexusNetworkHtb(configs) {
                 consent_required: gdprStatus.applies,
                 consent_string: gdprStatus.consentString
             };
+
```

Comment:
This is just a FYI. For CCPA times out or the API is down, basically if we can't able to get CCPA data, we will return an empty string for uspString.